update_compilers/install_rust_compilers.sh

```diff
@@ -228,6 +228,9 @@ install_new_rust 1.37.0 RUST_TARGETS[@]
 install_new_rust 1.38.0 RUST_TARGETS[@]
 install_new_rust 1.39.0 RUST_TARGETS[@]
 install_new_rust 1.40.0 RUST_TARGETS[@]
+install_new_rust 1.41.0 RUST_TARGETS[@]
+RUST_TARGETS=( "${RUST_TARGETS[@]/i686-unknown-dragonfly/}" )
```

Comment:
Shouldn't this be set before the previous line?

Reply:
Thanks. Fixed.